orcoursetrion/lib/github.py

```diff
@@ -67,10 +72,37 @@ def _get_all(self, url):
             ):
                 response = self.session.get(response.links['next']['url'])
                 results += response.json()
-        if response.status_code != 200:
+        if response.status_code not in [200, 404]:
             raise GitHubUnknownError(response.text)
         return results
 
+    def _get_repo(self, org, repo):
+        """Either return the repo dictionary, or None if it doesn't exists.
+
+        Args:
+            org (str): Organization the repo lives in.
+            repo (str): The name of the repo.
+        Raises:
+            requests.exceptions.RequestException
+            GitHubUnknownError
```

Comment:
Alright, cool. So all is well. It might make the docs look a tiny bit more consistent to have absolute paths to these exceptions everywhere, so I'll add that as an issue as you suggested. Definitely low priority.